decisions/13-schemaorg-identifier-as-PropertyValue.md

```diff
@@ -58,4 +60,4 @@ is the same as
 }
```

Comment:
In the third example, is the intent really to assert that the graph node of type `identifier` has `id` of `"https://doi.org/10.1234/56789"`? Seems that `id` value should really be on the Dataset class.

Reply:
While I renamed the decision file for `13-schemaorg-identifier-as-PropertyValue.md` in this PR, I think we should move substantive discussion of its content to another issue or PR. Probably in issue #13.

Reply:
Copied comment to issue #13